astropy/time/tests/test_basic.py

```diff
@@ -433,7 +433,7 @@ def test_datetime(self):
         assert t.datetime == datetime.datetime(2000, 1, 1, 1, 1, 1, 123457)
 
         # broadcasting
-        dt3 = (dt + (dt2-dt)*np.arange(12)).reshape(4, 3)
+        dt3 = (dt + (dt2 - dt) * np.arange(12)).reshape(4, 3)
```

Comment:
Most changes in the test file are fine, but here I'd not do the spaces around `-` (but do the ones around `*`)